ffi/derive/tests/ui_fail/private_method_inside_impl.rs

```diff
@@ -0,0 +1,19 @@
+use std::alloc::alloc;
```

Comment:
Like I said previously, just skip private functions and (optionally) warn about them. I'm quite sure that `cargo lints` turns all warnings into errors, so notifying of private associated functions might not be possible.